cmd/minikube/cmd/start.go

```diff
@@ -1210,3 +1202,10 @@ func getKubernetesVersion(old *config.ClusterConfig) string {
 
 	return version.VersionPrefix + nvs.String()
 }
+
+func skipErrorIfForceEnabled(code int, message string, v out.V) {
+	if !viper.GetBool(force) {
+		exit.WithCodeT(code, message, v)
```

Comment:
hint to the user if they really want this, they can use --force flag
similar to what we had before
```
To use this version anyway, use the `--force` flag
```

Reply:
I don't think we should be encouraging people to just use the `--force` flag if something isn't meeting minimum requirements -- it makes sense for k8s versions because users may be using <1.13 which is a reasonable use case, but not for anything else